consumer/src/extractors/json_token_extractor.e

```diff
@@ -24,7 +24,12 @@ feature -- Extractor
 				end
 				if attached {JSON_STRING} l_object.item ("refresh_token") as l_value then
 					if attached Result as l_result then
-							l_result.set_refresh_token (l_value.item)
+						l_result.set_refresh_token (l_value.item)
+					end
+				end
+				if attached {JSON_NUMBER} l_object.item ("expires_in") as l_value then
+					if attached Result as l_result and then l_value.is_integer then
```

Comment:
Result is already a local
so replace
   if attached Result as l_result ..
by
  if Result /= Void ..